doc/nrf/releases_and_maturity/migration/migration_hwmv2.rst

```diff
@@ -22,11 +22,15 @@ The current hardware model adds support for the following features:
 * Support for advanced use cases with :ref:`zephyr:sysbuild`.
 * Removal of all existing arbitrary and inconsistent uses of Kconfig and folder names.
 
-For more information about the rationale, development, and concepts behind the new model, see the `Zephyr original issue <Zephyr issue #51831>`_, the `Zephyr original Pull Request <Zephyr pull request #50305>`_,.
-For the complete set of changes introduced, consult the `hardware model v2 commit <Zephyr commit 8dc3f8>`_.
+For more information about the rationale, development, and concepts behind the new model, see the following links:
+
+* Zephyr original issue: `Zephyr issue #51831`_
+* Zephyr original pull request: `Zephyr pull request #50305`_
+
+For the complete set of changes introduced, consult the hardware model v2 commit: `Zephyr commit 8dc3f8`_.
```

Comment:
Can you point to this migration guide from the note in https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/config_and_build/board_support/board_names.html :
![image](https://github.com/nrfconnect/sdk-nrf/assets/41291385/3ee067ad-128e-469d-91e8-cf899a33cfc6)
The marked sentence would be better off with links to our internal migration guide than Zephyr.